mobile/native/src/ln_dlc/mod.rs

```diff
@@ -910,6 +910,8 @@ fn update_state_after_collab_revert(
                 order_expiry_timestamp: OffsetDateTime::now_utc(),
                 reason: OrderReason::Expired,
                 stable: position.stable,
+                // TODO(bonomat): why do we create an order here if we filling failed?
+                failure_reason: None,
```

Comment:
I guess so that we can continue with the rest of this function since we need an `Order`. Did you write this, @holzeis?